wagtail/admin/api/actions/create_alias.py

```diff
@@ -0,0 +1,52 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.api.v2.utils import BadRequestError
+from wagtail.core.actions.create_alias import CreatePageAliasAction, CreatePageAliasIntegrityError
+from wagtail.core.models import Page
+
+from .base import APIAction
+
+
+class CreatePageAliasAPIActionSerializer(Serializer):
+    parent_page_id = fields.IntegerField(required=False)
+    recursive = fields.BooleanField(default=False, required=False)
+    update_slug = fields.CharField(required=False)
```

Comment:
Wondering if we should rename this to ``slug`` since it does the exact same job as ``slug`` in the copy page API?

I do slightly prefer naming it ``update_slug`` in both places though! What do you think?

Reply:
I don't have a strong opinion on this. Since you prefer `update_slug` we can leave it here and I can change it in the copy page API.